coala_quickstart/generation/Settings.py

```diff
@@ -78,6 +79,8 @@ def generate_settings(project_dir, project_files, ignore_globs, relevant_bears):
         The list of ignore glob expressions.
     :param relevant_bears:
         A dict with language name as key and bear classes as value.
+    :param incomplete_sections:
+        Whether coala-quickstart is in allow-incomplete-settings mode.
```

Comment:
No need to mention `coala-quickstart`.  This is just an argument of function, which could be used in many ways. ;)

Describe the effects of enabling this within this function, not within the program.